.nvmrc

```diff
@@ -0,0 +1 @@
+v12.16.3
```

Comment:
Para cambiar de version de node specifica.
Para mas información:
[nvm for linux](https://github.com/nvm-sh/nvm)
[nvm for windows](https://github.com/coreybutler/nvm-windows)